README.adoc

```diff
@@ -2185,6 +2185,28 @@ File.open('testfile') do |f|
 end
 ----
 
+Atomic File Operations [[atomic-file-operations]]
```

Comment:
Seems to me you forgot the heading marker here (`===`).

Reply:
Oops, I added `===`.